include/aws/common/byte_buf.h

```diff
@@ -651,7 +667,18 @@ AWS_COMMON_API bool aws_byte_cursor_read_float_be64(struct aws_byte_cursor *cur,
  * If there is insufficient space in the cursor, returns false, leaving the
  * cursor unchanged.
  */
-AWS_COMMON_API bool aws_byte_cursor_read_be64(struct aws_byte_cursor *cur, uint64_t *var);
+AWS_COMMON_API bool aws_byte_cursor_read_float_be64(struct aws_byte_cursor *cur, double *var);
```

Comment:
Maybe it's better to rename it as `read_double_be64`

Reply:
I didn't rename this. I just moved it so all the `read_float` functions were together

Reply:
I understand, what I meant was "It might be better to just rename it." because the argument itself has double type.